entity/src/vulnerability.rs

```diff
@@ -7,6 +7,8 @@ pub struct Model {
     #[sea_orm(primary_key)]
     pub id: i32,
     pub identifier: String,
+    pub title: Option<String>,
+    pub description_en: Option<String>,
```

Comment:
The `_en` feels wrong as a field name suffix. Feels hard-coded. Would it make more sense as a LANG env var?

Reply:
Fwiw it's lang from the CVE not user.

Reply:
But the `"en"` is hardcoded in the filter below. If it's from the CVE, does that imply an additional `pub lang: String,` field? That also feels a little overkilly to me, but if we need to support multiple langs...

Reply:
>  is hardcoded in the filter below
good catch!

Reply:
Yes I reckon there could be multiple because yes there is a lang property. 

But it could be anything or multiple I reckon.